matecheck.py

```diff
@@ -242,6 +275,8 @@ def analyze_fens(self, fens):
     print("Total FENs:   ", numfen)
     print("Found mates:  ", mates)
     print("Best mates:   ", bestmates)
+    print("Found mates nodes searched:", totalnodes)
```

Comment:
is the total most interesting, or the average and maximum?

Reply:
I thought avg can be computed from total and `Found mates`.

Reply:
Adding max is a good idea, maybe also min. but both will depend on `bm` of course....